app/lib/methods/helpers/theme.ts

```diff
@@ -47,19 +44,6 @@ export const newThemeState = (prevState: { themePreferences: IThemePreference },
 	return { themePreferences, theme: getTheme(themePreferences) };
 };
 
-export const setNativeTheme = async (themePreferences: IThemePreference): Promise<void> => {
-	const theme = getTheme(themePreferences);
-	if (isAndroid) {
-		const iconsLight = theme === 'light';
-		try {
-			// The late param as default is true @ react-native-navigation-bar-color/src/index.js line 8
-			await changeNavigationBarColor(themes[theme].surfaceLight, iconsLight, true);
-		} catch (error) {
-			// Do nothing
-		}
-	}
-	setRootViewColor(themes[theme].surfaceRoom);
```

Comment:
We still need to call this.
You can see it on iOS while rotating your device portrait <-> landscape.
Without it, it should be a white flash.